terraform/aks/config/production.tfvars.json

```diff
@@ -0,0 +1,21 @@
+{
+  "cluster": "production",
+  "cluster_rg": "s189p01-tsc-pd-rg",
+  "cluster_name": "s189p01-tsc-production-aks",
+  "namespace": "tra-production",
+  "environment": "production",
+  "deploy_azure_backing_services": true,
+  "key_vault_name": "s189p01-ccbl-pd-kv",
+  "key_vault_app_secret_name": "CCBL-APP-SECRETS-PRODUCTION",
+  "key_vault_infra_secret_name": "CCBL-INFRA-SECRETS-PRODUCTION",
+  "enable_monitoring": true,
+  "external_url": "https://check-the-childrens-barred-list.education.gov.uk/healthcheck",
+  "statuscake_contact_groups": [249142],
+  "startup_command": [
+    "/bin/sh",
+    "-c",
+    "bundle exec rails db:migrate && bundle exec rails server -b 0.0.0.0"
+  ],
+  "gov_uk_host_names": ["check-the-childrens-barred-list.education.gov.uk"],
+  "azure_enable_backup_storage": true
+}
```

Comment:
For postgres, I would add a maintenance window for prod, which forces any pg maintenance to a specific window e.g.
  "azure_maintenance_window": {
    "day_of_week": 0,
    "start_hour": 3,
    "start_minute": 0
  }
and decide if you need high availability, set with
  "postgres_enable_high_availability": true